### PR TITLE
Extension reviews

### PR DESCRIPTION
**Pull request by PlanetEfficacy**

### CODE REVIEW

**Comment by PlanetEfficacy** [comment]
Please hold off on this PR. A test is failing.

**Comment by PlanetEfficacy** [comment]
This ready to merge.

File: app/views/items/show.html.erb [hunk 1]
```diff
@@ -1,11 +1,80 @@
...
-<%= number_to_currency(@item.dollars) %>
-<%= image_tag @item.product_image, class: "product-image" %>
-<%#image_tag @item.image.url(:medium), class: "product-image" %>
-
-<% if @item.retired? %>
-  <div class="label label-warning">Item Retired</div>
-<% else %>
-  <%= button_to "Add to Cart", cart_path(item_id: @item.id) %>
-<% end %>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
```
Comment (susiirwin) [inline app/views/items/show.html.erb]: Is this absolutely necessary? We have the gem, so this seems superfluous, but we should investigate a bit.

**Comment by susiirwin** [comment]
This is ridiculously cool!!! Nice work!! 👍

**Pull request merged**